addons/mail/static/tests/helpers/mock_server.js

```diff
@@ -50,7 +50,50 @@ MockServer.include({
             return channel;
         });
     },
-
+    /**
+     * Simulate the 'channel_fold' route of of mail.channel
+     * In particular sends a notification on the bus
+     *
+     * @private
+     */
+     _mockChannelFold(args) {
+        const { state, uuid } = args.kwargs;
+        // Should use something of the form session.partner_id or env.messaging.currentPartner
+        // This is a limitation of the architecture of JS tests
+        const partnerId = undefined;
```

Comment:
Useless variable, just explicitly pass `undefined` were needed and move comment above next to where explicit `undefined` is passed.